components/math/multiplicationConnect/GameBoard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+
+const createGrid = () => {
+    let arr = [];
+    for(let i=1; i<36; i++){
```

Comment:
https://stackoverflow.com/questions/3746725/how-to-create-an-array-containing-1-n

Reply:
is it necessary to refactor this? I'm getting errors trying to use the following:

```
return Array.from(Array(35), (e,i)=>i+1).map(i => {
            return {
              key: i,
              gridNumber: i,
            }
          });
```

Reply:
something about React children not being able to have an object and to use an array instead